test/recipes/03-test_fipsinstall.t

```diff
@@ -262,42 +286,55 @@ ok(replace_parent_line_file('fips_no_module_mac.cnf',
                 '-config', 'fips_parent_no_module_mac.cnf'])),
    "verify load config fail no module mac");
 
-ok(replace_parent_line_file('fips_no_install_mac.cnf',
-                            'fips_parent_no_install_mac.cnf')
-   && !run(app(['openssl', 'fipsinstall',
-                '-config', 'fips_parent_no_install_mac.cnf'])),
-   "verify load config fail no install mac");
-
-ok(replace_parent_line_file('fips_bad_indicator.cnf',
-                            'fips_parent_bad_indicator.cnf')
-   && !run(app(['openssl', 'fipsinstall',
-                '-config', 'fips_parent_bad_indicator.cnf'])),
-   "verify load config fail bad indicator");
-
-
-ok(replace_parent_line_file('fips_bad_install_mac.cnf',
-                            'fips_parent_bad_install_mac.cnf')
-   && !run(app(['openssl', 'fipsinstall',
-                '-config', 'fips_parent_bad_install_mac.cnf'])),
-   "verify load config fail bad install mac");
+SKIP: {
```

Comment:
I'm unsure as to supporting this here.  This and the modified block further down is verifying that the self check on install works.  This is removed in 140-3, should these tests also be removed?

I'm leaning towards _yes_ to avoid clutter here.  This would also let us remove some of the config support for the setting.

Reply:
Depends.. Is the code related to running once going to be removed from everywhere (inside the fipsinstall app also I mean).

Reply:
I'd think so.  That code is dead currently.

Reply:
I think that should be removed then at the same time (And the docs updated)